schema/system/consistency_checks.json

```diff
@@ -0,0 +1,51 @@
+{
+    "$id": "system/consistency-checks",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "title": "consistency checks",
+    "properties": {
+        "keys": {
+            "type": "array",
```

Comment:
We need to describe what these are

Reply:
Might look cumbersome, but outlines the idea: "The output of consistency checks performed on data adhering to JSON schema, but inconsistent with scientific or logical rules, to show problems in UI."